src/help/zaphelp/contents/ui/dialogs/options/checkforupdates.html

```diff
@@ -46,8 +46,8 @@
 They may be incomplete, contain significant issues or cause stability problems.
 
 <H3>Add-on directories</H3>
-ZAP will load all of the add-ons located in the release 'plugins' directory as well as any that have been downloaded
-and are in the local 'plugins' directory.<br>
+ZAP will load all of the add-ons located in the release 'plugin' directory as well as any that have been downloaded
+and are in the local 'plugin' directory.<br>
```

Comment:
Should the terminology here be the same as above? Zap and user vs release and local?

<seb>Edited: Comment moved to PR.</sub>